extension/src/main/kotlin/CamundaRestClientSpringBootExtension.kt

```diff
@@ -46,7 +49,7 @@ import org.springframework.context.annotation.Import
  * Basic configuration of the extension.
  */
 @Configuration
-@ComponentScan
+@Import(RemoteExternalTaskService::class, RemoteRepositoryService::class, RemoteRuntimeService::class, RemoteTaskService::class)
```

Comment:
This import is not sufficient, since `FeignErroDecoderConfiguration` is missing. Long term we will end up in 13 Camunda Components and some additions... Is it really the way to go?

Reply:
One could always delegate importing to sub-configuration classes, in order to split up the definition. 🤔